webdriver-spec.html

```diff
@@ -2614,26 +2618,16 @@
    <a>URL prefix</a> of the <a>upstream</a> <a>remote end</a> will need
    to be tracked.
 
- <li><p>If the <a>maximum active sessions</a> is equal to
-  the length of the list of <a>active sessions</a>,
-  return <a>error</a> with <a>error code</a> <a>session not created</a>.
-
  <li><p>If there is a <a>current user prompt</a>,
   return <a>error</a> with <a>error code</a> <a>session not created</a>.
 
  <li><p>Let <var>capabilities</var> be the result
   of <a>trying</a> to <a>process capabilities</a>
   with <var>parameters</var> as an argument.
 
- <li><p>If <var>capabilities result</var> is an <a>error</a>,
-  return <a>error</a> with <a>error code</a> <a>session not created</a> and
-  the error data from <var>capabilities result</var>.
-
- <li><p>If <var>capabilities result</var>'s data is <a>null</a>,
+ <li><p>If <var>capabilities</var>'s data is <a>null</a>,
```

Comment:
`Trying` will either have returned an error to the user, or will return the `data` field. This should be `If capabilities is null`